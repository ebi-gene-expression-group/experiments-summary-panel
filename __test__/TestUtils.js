import URI from 'urijs'

const bulkExperiments = [
  {
    experimentType: `MICROARRAY_ANY`,
    experimentAccession: `E-GEOD-40611`,
    experimentDescription: `Transcription profiling by array of parotid tissue from Primary Sjögren’s Syndrome and controls`,
    lastUpdate: `06-03-2019`,
    numberOfAssays: 49,
    numberOfContrasts: 2,
    species: `Homo sapiens`,
    kingdom: `animals`,
    experimentalFactors: [
      `block`,
      `disease`
    ],
    arrayDesigns: [
      `A-AFFY-44`
    ],
    arrayDesignNames: [
      `Affymetrix GeneChip Human Genome U133 Plus 2.0 [HG-U133_Plus_2]`
    ]
  },
  {
    experimentType: `RNASEQ_MRNA_BASELINE`,
    experimentAccession: `E-MTAB-451`,
    experimentDescription: `RNA-Seq of Schistosoma mansoni (flatworms) larva and adult individuals at different life-stages`,
    lastUpdate: `06-10-2018`,
    numberOfAssays: 11,
    numberOfContrasts: 0,
    species: `Schistosoma mansoni`,
    kingdom: `animals`,
    experimentalFactors: [
      `developmental stage`
    ],
    arrayDesigns: [],
    arrayDesignNames: []
  },
  {
    experimentType: `RNASEQ_MRNA_BASELINE`,
    experimentAccession: `E-MTAB-2836`,
    experimentDescription: `RNA-seq of coding RNA from tissue samples of 122 human individuals representing 32 different tissues`,
    lastUpdate: `05-10-2018`,
    numberOfAssays: 200,
    numberOfContrasts: 0,
    species: `Homo sapiens`,
    kingdom: `animals`,
    experimentalFactors: [
      `organism part`
    ],
    arrayDesigns: [],
    arrayDesignNames: []
  },
  {
    experimentType: `RNASEQ_MRNA_BASELINE`,
    experimentAccession: `E-MTAB-4401`,
    experimentDescription: `Transcription profiling by high throughput sequencing of different tissues from Brachypodium distachyon (Bd21 inbred line)`,
    lastUpdate: `05-10-2018`,
    numberOfAssays: 11,
    numberOfContrasts: 0,
    species: `Brachypodium distachyon`,
    kingdom: `plants`,
    experimentalFactors: [
      `organism part`
    ],
    arrayDesigns: [],
    arrayDesignNames: []
  },
  {
    experimentType: `RNASEQ_MRNA_DIFFERENTIAL`,
    experimentAccession: `E-MTAB-1913`,
    experimentDescription: `Salt stress in salt sensitive Oryza sativa Indica group IR29 `,
    lastUpdate: `04-10-2018`,
    numberOfAssays: 30,
    numberOfContrasts: 15,
    species: `Oryza sativa Indica group`,
    kingdom: `plants`,
    experimentalFactors: [
      `growth condition`,
      `time`
    ],
    arrayDesigns: [],
    arrayDesignNames: []
  },
  {
    experimentType: `MICROARRAY_ANY`,
    experimentAccession: `E-MAXD-6`,
    experimentDescription: `Transcription profiling by array of Drosophila larvae after parasitoid attack`,
    lastUpdate: `04-10-2018`,
    numberOfAssays: 89,
    numberOfContrasts: 9,
    species: `Drosophila melanogaster`,
    kingdom: `animals`,
    experimentalFactors: [
      `infect`,
      `time`
    ],
    arrayDesigns: [
      `A-AFFY-17`
    ],
    arrayDesignNames: [
      `Affymetrix GeneChip Drosophila Genome [DrosGenome1]`
    ]
  },
  {
    experimentType: `MICROARRAY_ANY`,
    experimentAccession: `E-MEXP-1810`,
    experimentDescription: `Transcription profiling by array of C. elegans isolates treated with dauer larva-inducing pheromone`,
    lastUpdate: `25-09-2018`,
    numberOfAssays: 12,
    numberOfContrasts: 2,
    species: `Caenorhabditis elegans`,
    kingdom: `animals`,
    experimentalFactors: [
      `compound`,
      `strain`
    ],
    arrayDesigns: [
      `A-AFFY-60`
    ],
    arrayDesignNames: [
      `Affymetrix GeneChip C. elegans Genome Array [Celegans]`
    ]
  },
  {
    experimentType: `MICROARRAY_ANY`,
    experimentAccession: `E-GEOD-43049`,
    experimentDescription: `Caco-2 cells: cultured in conventional vs apical anaerobic conditions`,
    lastUpdate: `25-09-2018`,
    numberOfAssays: 12,
    numberOfContrasts: 1,
    species: `Homo sapiens`,
    kingdom: `animals`,
    experimentalFactors: [
      `growth condition`
    ],
    arrayDesigns: [
      `A-AGIL-28`
    ],
    arrayDesignNames: [
      `Agilent Whole Human Genome Microarray 4x44K 014850 G4112F (85 cols x 532 rows)`
    ]
  }
]

const singleCellExperiments = [
  {
    experimentType: `SINGLE_CELL_RNASEQ_MRNA_BASELINE`,
    experimentAccession: `E-EHCA-2`,
    experimentDescription: `Melanoma infiltration of stromal and immune cells`,
    lastUpdate: `16-11-2018`,
    numberOfAssays: 6638,
    numberOfContrasts: 0,
    species: `Mus musculus`,
    kingdom: `animals`,
    experimentalFactors: [
      `single cell identifier`,
      `sampling site`,
      `time`
    ],
    arrayDesigns: [],
    arrayDesignNames: []
  },
  {
    experimentType: `SINGLE_CELL_RNASEQ_MRNA_BASELINE`,
    experimentAccession: `E-MTAB-5061`,
    experimentDescription: `Single-cell RNA-seq analysis of human pancreas from healthy individuals and type 2 diabetes patients`,
    lastUpdate: `11-10-2018`,
    numberOfAssays: 3514,
    numberOfContrasts: 0,
    species: `Homo sapiens`,
    kingdom: `animals`,
    experimentalFactors: [
      `single cell identifier`,
      `disease`
    ],
    arrayDesigns: [],
    arrayDesignNames: []
  },
  {
    experimentType: `SINGLE_CELL_RNASEQ_MRNA_BASELINE`,
    experimentAccession: `E-GEOD-99058`,
    experimentDescription: `Single cell RNA-seq of mouse brain astrocyte transcriptomes`,
    lastUpdate: `11-10-2018`,
    numberOfAssays: 250,
    numberOfContrasts: 0,
    species: `Mus musculus`,
    kingdom: `animals`,
    experimentalFactors: [
      `single cell identifier`
    ],
    arrayDesigns: [],
    arrayDesignNames: []
  }
]

const buildFeaturedExperimentsCards = (host) => {
  const linkToImage = (imgFileName) =>
    URI(`resources/images/experiments-summary/${imgFileName}.png`, host).toString()
  const linkToExperiment = (accession) =>
    URI(`experiments/${accession}`, host).toString()
  const linkToExperimentSet = (keyword) =>
    URI(`/experiments`, host).search({experimentSet: keyword}).toString()

  return [
    {
      iconType: `image`,
      iconSrc: linkToImage(`encode`),
      description: {
      //   text: ``,
        url: linkToExperimentSet(`ENCODE`)
      },
      content: [
        {
          text: `Human tissues`,
          url: linkToExperiment(`E-MTAB-4344`)
        },
        {
          text: `Human cells`,
          url: linkToExperiment(`E-GEOD-26284`)
        }
      ]
    },
    {
      iconType: `image`,
      iconSrc: linkToImage(`blueprint`),
      description: {
      //   text: ``,
        url: linkToExperimentSet(`BLUEPRINT`)
      },
      content: [
        {
          text: `Plasma cells of tonsil`,
          url: linkToExperiment(`E-MTAB-4754`)
        },
        {
          text: `Rare types of haemopoetic cells`,
          url: linkToExperiment(`E-MTAB-3819`)
        },
        {
          text: `Common types of haemopoetic cells`,
          url: linkToExperiment(`E-MTAB-3827`)
        }
      ]
    },
    {
      iconType: `image`,
      iconSrc: linkToImage(`fantom`),
      description: {
      //   text: ``,
        url: linkToExperimentSet(`FANTOM5`)
      },
      content: [
        {
          text: `Mouse cells`,
          url: linkToExperiment(`E-MTAB-3578`)
        },
        {
          text: `Mouse tissues`,
          url: linkToExperiment(`E-MTAB-3579`)
        },
        {
          text: `Human tissues`,
          url: linkToExperiment(`E-MTAB-3358`)
        }
      ]
    },
    {
      iconType: `image`,
      iconSrc: linkToImage(`human_protein_atlas`),
      description: {
      //   text: ``,
        url: linkToExperiment(`E-PROT-3`)
      },
      content: [
        {
          text: `Human tissues`,
          url: linkToExperiment(`E-PROT-3`)
        }
      ]
    },
    {
      iconType: `image`,
      iconSrc: linkToImage(`ccle`),
      description: {
      //   text: ``,
        url: linkToExperiment(`E-MTAB-2770`)
      },
      content: [
        {
          text: `Cancer Cell Line Encyclopedia`,
          url: linkToExperiment(`E-MTAB-2770`)
        }
      ]
    },
    {
      iconType: `image`,
      iconSrc: linkToImage(`hipsci`),
      description: {
      //   text: ``,
        url: linkToExperimentSet(`HipSci`)
      },
      content: [
        {
          text: `Proteomics – Cell lines`,
          url: linkToExperiment(`E-PROT-5`)
        },
        {
          text: `RNA – Cell lines`,
          url: linkToExperiment(`E-MTAB-4748`)
        }
      ]
    },
    {
      iconType: `image`,
      iconSrc: linkToImage(`gtex`),
      description: {
      //   text: ``,
        url: linkToExperiment(`E-MTAB-5214`)
      },
      content: [
        {
          text: `Human tissues`,
          url: linkToExperiment(`E-MTAB-5214`)
        }
      ]
    },
    {
      iconType: `image`,
      iconSrc: linkToImage(`pcawg`),
      description: {
      //   text: ``,
        url: linkToExperimentSet(`Pan-Cancer`)
      },
      content: [
        {
          text: `PCAWG by disease`,
          url: linkToExperiment(`E-MTAB-5200`)
        },
        {
          text: `PCAWG by individual`,
          url: linkToExperiment(`E-MTAB-5423`)
        }
      ]
    },
    {
      iconType: `image`,
      iconSrc: linkToImage(`wtsi_mgh_cancerrxgene`),
      description: {
      //   text: ``,
        url: linkToExperiment(`E-MTAB-3983`)
      },
      content: [
        {
          text: `Genomics of Drug Sensitivity in Cancer Project – Cell lines`,
          url: linkToExperiment(`E-MTAB-3983`)
        }
      ]
    },
    {
      iconType: `image`,
      iconSrc: linkToImage(`hdbr`),
      description: {
      //   text: ``,
        url: linkToExperiment(`E-MTAB-4840`)
      },
      content: [
        {
          text: `Prenatal brain development`,
          url: linkToExperiment(`E-MTAB-4840`)
        }
      ]
    },
    {
      iconType: `image`,
      iconSrc: linkToImage(`baseline`),
      description: {
      //   text: ``,
        url: URI(`baseline/experiments`, host).toString()
      },
      content: [
        {
          text: `Baseline experiments`,
          url: URI(`baseline/experiments`, host).toString()
        }
      ]
    },
    {
      iconType: `image`,
      iconSrc: linkToImage(`gramene`),
      description: {
      //   text: ``,
        url: URI(`plant/experiments`, host).toString()
      },
      content: [
        {
          text: `Plant experiments`,
          url: URI(`plant/experiments`, host).toString()
        }
      ]
    }
  ]
}

export { bulkExperiments, singleCellExperiments, buildFeaturedExperimentsCards }
