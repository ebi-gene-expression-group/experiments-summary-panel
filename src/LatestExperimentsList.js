import React from 'react'
import PropTypes from 'prop-types'

import LatestExperimentEntry from './LatestExperimentEntry'
import { experimentPropTypes } from './experimentsPropTypes'

function LatestExperimentsList ({ experiments, host }) {
  return (
    <ul style={{ listStyle: `none`, marginLeft: `offset` }}>
      {
        Array.isArray(experiments) &&
        experiments.map((experiment, idx) =>
          <li key={idx}>
            <LatestExperimentEntry host={host} {...experiment}/>
          </li>
        )
      }
    </ul>
  )
}

LatestExperimentsList.propTypes = {
  experiments: PropTypes.arrayOf(PropTypes.shape(experimentPropTypes)).isRequired,
  host: PropTypes.string.isRequired
}

export default LatestExperimentsList
