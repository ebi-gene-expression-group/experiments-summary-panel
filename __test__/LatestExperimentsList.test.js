import React from 'react'
import renderer from 'react-test-renderer'

import { bulkExperiments, singleCellExperiments } from './TestUtils'

import LatestExperimentsList from '../src/LatestExperimentsList'

describe(`LatestExperimentsList`, () => {
  test.each([
    [`latest bulk experiments`, bulkExperiments],
    [`latest single cell experiments`, singleCellExperiments]
  ])(`matches snapshot: %s`, (titleText, experiments) => {
    const tree = renderer
      .create(<LatestExperimentsList
        host={`https://www.ebi.ac.uk/gxa/`}
        experiments={experiments}
      />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
