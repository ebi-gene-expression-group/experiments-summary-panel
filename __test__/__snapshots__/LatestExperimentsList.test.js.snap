// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LatestExperimentsList matches snapshot: latest bulk experiments 1`] = `
<ul
  style={
    Object {
      "listStyle": "none",
      "marginLeft": "offset",
    }
  }
>
  <li>
    <div
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
        }
      }
    >
      <div
        className="sc-bczRLJ iuLMzT"
      >
        <span
          className="sc-gsnTZi dsXvwY button"
        >
          Differential
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <span
          className="sc-gsnTZi iupGoc button"
          style={
            Object {
              "minWidth": "120px",
            }
          }
          title="Number of assays in experiment"
        >
          49
           assays
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <a
          className="button"
          href="https://www.ebi.ac.uk/gxa/experiments/E-GEOD-40611"
          style={
            Object {
              "backgroundColor": "#3497C5",
              "marginBottom": "0px",
            }
          }
        >
          Results
        </a>
      </div>
      <div
        className="sc-bczRLJ iuLMzT"
        style={
          Object {
            "paddingLeft": "1rem",
          }
        }
      >
        <small>
          06-03-2019
        </small>
        <br />
        <a
          href="https://www.ebi.ac.uk/gxa/experiments/E-GEOD-40611"
        >
          Transcription profiling by array of parotid tissue from Primary Sjögren’s Syndrome and controls
           – 
          <i>
            Homo sapiens
          </i>
        </a>
      </div>
    </div>
  </li>
  <li>
    <div
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
        }
      }
    >
      <div
        className="sc-bczRLJ iuLMzT"
      >
        <span
          className="sc-gsnTZi dsXvwY button"
        >
          Baseline
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <span
          className="sc-gsnTZi iupGoc button"
          style={
            Object {
              "minWidth": "120px",
            }
          }
          title="Number of assays in experiment"
        >
          11
           assays
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <a
          className="button"
          href="https://www.ebi.ac.uk/gxa/experiments/E-MTAB-451"
          style={
            Object {
              "backgroundColor": "#3497C5",
              "marginBottom": "0px",
            }
          }
        >
          Results
        </a>
      </div>
      <div
        className="sc-bczRLJ iuLMzT"
        style={
          Object {
            "paddingLeft": "1rem",
          }
        }
      >
        <small>
          06-10-2018
        </small>
        <br />
        <a
          href="https://www.ebi.ac.uk/gxa/experiments/E-MTAB-451"
        >
          RNA-Seq of Schistosoma mansoni (flatworms) larva and adult individuals at different life-stages
           – 
          <i>
            Schistosoma mansoni
          </i>
        </a>
      </div>
    </div>
  </li>
  <li>
    <div
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
        }
      }
    >
      <div
        className="sc-bczRLJ iuLMzT"
      >
        <span
          className="sc-gsnTZi dsXvwY button"
        >
          Baseline
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <span
          className="sc-gsnTZi iupGoc button"
          style={
            Object {
              "minWidth": "120px",
            }
          }
          title="Number of assays in experiment"
        >
          200
           assays
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <a
          className="button"
          href="https://www.ebi.ac.uk/gxa/experiments/E-MTAB-2836"
          style={
            Object {
              "backgroundColor": "#3497C5",
              "marginBottom": "0px",
            }
          }
        >
          Results
        </a>
      </div>
      <div
        className="sc-bczRLJ iuLMzT"
        style={
          Object {
            "paddingLeft": "1rem",
          }
        }
      >
        <small>
          05-10-2018
        </small>
        <br />
        <a
          href="https://www.ebi.ac.uk/gxa/experiments/E-MTAB-2836"
        >
          RNA-seq of coding RNA from tissue samples of 122 human individuals representing 32 different tissues
           – 
          <i>
            Homo sapiens
          </i>
        </a>
      </div>
    </div>
  </li>
  <li>
    <div
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
        }
      }
    >
      <div
        className="sc-bczRLJ iuLMzT"
      >
        <span
          className="sc-gsnTZi dsXvwY button"
        >
          Baseline
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <span
          className="sc-gsnTZi iupGoc button"
          style={
            Object {
              "minWidth": "120px",
            }
          }
          title="Number of assays in experiment"
        >
          11
           assays
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <a
          className="button"
          href="https://www.ebi.ac.uk/gxa/experiments/E-MTAB-4401"
          style={
            Object {
              "backgroundColor": "#3497C5",
              "marginBottom": "0px",
            }
          }
        >
          Results
        </a>
      </div>
      <div
        className="sc-bczRLJ iuLMzT"
        style={
          Object {
            "paddingLeft": "1rem",
          }
        }
      >
        <small>
          05-10-2018
        </small>
        <br />
        <a
          href="https://www.ebi.ac.uk/gxa/experiments/E-MTAB-4401"
        >
          Transcription profiling by high throughput sequencing of different tissues from Brachypodium distachyon (Bd21 inbred line)
           – 
          <i>
            Brachypodium distachyon
          </i>
        </a>
      </div>
    </div>
  </li>
  <li>
    <div
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
        }
      }
    >
      <div
        className="sc-bczRLJ iuLMzT"
      >
        <span
          className="sc-gsnTZi dsXvwY button"
        >
          Differential
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <span
          className="sc-gsnTZi iupGoc button"
          style={
            Object {
              "minWidth": "120px",
            }
          }
          title="Number of assays in experiment"
        >
          30
           assays
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <a
          className="button"
          href="https://www.ebi.ac.uk/gxa/experiments/E-MTAB-1913"
          style={
            Object {
              "backgroundColor": "#3497C5",
              "marginBottom": "0px",
            }
          }
        >
          Results
        </a>
      </div>
      <div
        className="sc-bczRLJ iuLMzT"
        style={
          Object {
            "paddingLeft": "1rem",
          }
        }
      >
        <small>
          04-10-2018
        </small>
        <br />
        <a
          href="https://www.ebi.ac.uk/gxa/experiments/E-MTAB-1913"
        >
          Salt stress in salt sensitive Oryza sativa Indica group IR29 
           – 
          <i>
            Oryza sativa Indica group
          </i>
        </a>
      </div>
    </div>
  </li>
  <li>
    <div
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
        }
      }
    >
      <div
        className="sc-bczRLJ iuLMzT"
      >
        <span
          className="sc-gsnTZi dsXvwY button"
        >
          Differential
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <span
          className="sc-gsnTZi iupGoc button"
          style={
            Object {
              "minWidth": "120px",
            }
          }
          title="Number of assays in experiment"
        >
          89
           assays
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <a
          className="button"
          href="https://www.ebi.ac.uk/gxa/experiments/E-MAXD-6"
          style={
            Object {
              "backgroundColor": "#3497C5",
              "marginBottom": "0px",
            }
          }
        >
          Results
        </a>
      </div>
      <div
        className="sc-bczRLJ iuLMzT"
        style={
          Object {
            "paddingLeft": "1rem",
          }
        }
      >
        <small>
          04-10-2018
        </small>
        <br />
        <a
          href="https://www.ebi.ac.uk/gxa/experiments/E-MAXD-6"
        >
          Transcription profiling by array of Drosophila larvae after parasitoid attack
           – 
          <i>
            Drosophila melanogaster
          </i>
        </a>
      </div>
    </div>
  </li>
  <li>
    <div
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
        }
      }
    >
      <div
        className="sc-bczRLJ iuLMzT"
      >
        <span
          className="sc-gsnTZi dsXvwY button"
        >
          Differential
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <span
          className="sc-gsnTZi iupGoc button"
          style={
            Object {
              "minWidth": "120px",
            }
          }
          title="Number of assays in experiment"
        >
          12
           assays
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <a
          className="button"
          href="https://www.ebi.ac.uk/gxa/experiments/E-MEXP-1810"
          style={
            Object {
              "backgroundColor": "#3497C5",
              "marginBottom": "0px",
            }
          }
        >
          Results
        </a>
      </div>
      <div
        className="sc-bczRLJ iuLMzT"
        style={
          Object {
            "paddingLeft": "1rem",
          }
        }
      >
        <small>
          25-09-2018
        </small>
        <br />
        <a
          href="https://www.ebi.ac.uk/gxa/experiments/E-MEXP-1810"
        >
          Transcription profiling by array of C. elegans isolates treated with dauer larva-inducing pheromone
           – 
          <i>
            Caenorhabditis elegans
          </i>
        </a>
      </div>
    </div>
  </li>
  <li>
    <div
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
        }
      }
    >
      <div
        className="sc-bczRLJ iuLMzT"
      >
        <span
          className="sc-gsnTZi dsXvwY button"
        >
          Differential
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <span
          className="sc-gsnTZi iupGoc button"
          style={
            Object {
              "minWidth": "120px",
            }
          }
          title="Number of assays in experiment"
        >
          12
           assays
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <a
          className="button"
          href="https://www.ebi.ac.uk/gxa/experiments/E-GEOD-43049"
          style={
            Object {
              "backgroundColor": "#3497C5",
              "marginBottom": "0px",
            }
          }
        >
          Results
        </a>
      </div>
      <div
        className="sc-bczRLJ iuLMzT"
        style={
          Object {
            "paddingLeft": "1rem",
          }
        }
      >
        <small>
          25-09-2018
        </small>
        <br />
        <a
          href="https://www.ebi.ac.uk/gxa/experiments/E-GEOD-43049"
        >
          Caco-2 cells: cultured in conventional vs apical anaerobic conditions
           – 
          <i>
            Homo sapiens
          </i>
        </a>
      </div>
    </div>
  </li>
</ul>
`;

exports[`LatestExperimentsList matches snapshot: latest single cell experiments 1`] = `
<ul
  style={
    Object {
      "listStyle": "none",
      "marginLeft": "offset",
    }
  }
>
  <li>
    <div
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
        }
      }
    >
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <span
          className="sc-gsnTZi iupGoc button"
          style={
            Object {
              "minWidth": "120px",
            }
          }
          title="Number of assays in experiment"
        >
          6,638
           assays
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <a
          className="button"
          href="https://www.ebi.ac.uk/gxa/experiments/E-EHCA-2"
          style={
            Object {
              "backgroundColor": "#3497C5",
              "marginBottom": "0px",
            }
          }
        >
          Results
        </a>
      </div>
      <div
        className="sc-bczRLJ iuLMzT"
        style={
          Object {
            "paddingLeft": "1rem",
          }
        }
      >
        <small>
          16-11-2018
        </small>
        <br />
        <a
          href="https://www.ebi.ac.uk/gxa/experiments/E-EHCA-2"
        >
          Melanoma infiltration of stromal and immune cells
           – 
          <i>
            Mus musculus
          </i>
        </a>
      </div>
    </div>
  </li>
  <li>
    <div
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
        }
      }
    >
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <span
          className="sc-gsnTZi iupGoc button"
          style={
            Object {
              "minWidth": "120px",
            }
          }
          title="Number of assays in experiment"
        >
          3,514
           assays
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <a
          className="button"
          href="https://www.ebi.ac.uk/gxa/experiments/E-MTAB-5061"
          style={
            Object {
              "backgroundColor": "#3497C5",
              "marginBottom": "0px",
            }
          }
        >
          Results
        </a>
      </div>
      <div
        className="sc-bczRLJ iuLMzT"
        style={
          Object {
            "paddingLeft": "1rem",
          }
        }
      >
        <small>
          11-10-2018
        </small>
        <br />
        <a
          href="https://www.ebi.ac.uk/gxa/experiments/E-MTAB-5061"
        >
          Single-cell RNA-seq analysis of human pancreas from healthy individuals and type 2 diabetes patients
           – 
          <i>
            Homo sapiens
          </i>
        </a>
      </div>
    </div>
  </li>
  <li>
    <div
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
        }
      }
    >
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <span
          className="sc-gsnTZi iupGoc button"
          style={
            Object {
              "minWidth": "120px",
            }
          }
          title="Number of assays in experiment"
        >
          250
           assays
        </span>
      </div>
      <div
        className="sc-bczRLJ iuLMzT hide-for-small-only"
      >
        <a
          className="button"
          href="https://www.ebi.ac.uk/gxa/experiments/E-GEOD-99058"
          style={
            Object {
              "backgroundColor": "#3497C5",
              "marginBottom": "0px",
            }
          }
        >
          Results
        </a>
      </div>
      <div
        className="sc-bczRLJ iuLMzT"
        style={
          Object {
            "paddingLeft": "1rem",
          }
        }
      >
        <small>
          11-10-2018
        </small>
        <br />
        <a
          href="https://www.ebi.ac.uk/gxa/experiments/E-GEOD-99058"
        >
          Single cell RNA-seq of mouse brain astrocyte transcriptomes
           – 
          <i>
            Mus musculus
          </i>
        </a>
      </div>
    </div>
  </li>
</ul>
`;
